Firestore/Example/Tests/Util/FSTHelpers.mm

```diff
@@ -385,32 +386,12 @@ - (nullable FSTQueryData *)queryDataForTarget:(TargetId)targetID {
   return aggregator.CreateRemoteEvent(doc.version);
 }
 
-FSTTargetChange *FSTTestTargetChangeMarkCurrent() {
-  return [[FSTTargetChange alloc] initWithResumeToken:[NSData data]
-      current:YES
-      addedDocuments:DocumentKeySet {}
-      modifiedDocuments:DocumentKeySet {}
-      removedDocuments:DocumentKeySet{}];
-}
-
-FSTTargetChange *FSTTestTargetChangeAckDocuments(DocumentKeySet docs) {
-  return [[FSTTargetChange alloc] initWithResumeToken:[NSData data]
-                                              current:YES
-                                       addedDocuments:docs
-                                    modifiedDocuments:DocumentKeySet {}
-                                     removedDocuments:DocumentKeySet{}];
-}
-
-FSTTargetChange *FSTTestTargetChange(DocumentKeySet added,
-                                     DocumentKeySet modified,
-                                     DocumentKeySet removed,
-                                     NSData *resumeToken,
-                                     BOOL current) {
-  return [[FSTTargetChange alloc] initWithResumeToken:resumeToken
-                                              current:current
-                                       addedDocuments:added
-                                    modifiedDocuments:modified
-                                     removedDocuments:removed];
+TargetChange FSTTestTargetChangeMarkCurrent() {
+  return {[NSData data], true, DocumentKeySet{}, DocumentKeySet{}, DocumentKeySet{}};
```

Comment:
Consider adding named parameters for the bool and DocKeySet params. Also immediately below. Optional.

(Possibly more useful for the one below. Here, I don't know which docKeySet is which, but it doesn't matter since they're all defaulted.)

Reply:
Done.